src/python/WMCore/Services/ReqMgr/ReqMgr.py

```diff
@@ -210,12 +210,18 @@ def getAbortedAndForceCompleteRequestsFromMemoryCache(self, expire=0):
         """
         _getAbortedAndForceCompleteRequestsFromMemoryCache_
         """
+
+
+        maskStates = ["aborted", "aborted-completed", "force-complete"]
+        return self.getRequestByStatusFromMemoryCache(maskStates, expire)
+
+    def getRequestByStatusFromMemoryCache(self, statusList, expire=0):
+
         # imports here to avoid the dependency not using this function
         from WMCore.Cache.GenericDataCache import MemoryCacheStruct
```

Comment:
can we move this import to the top of the module? I think it's better to have it all in the same place (even if it's not used all the time) than having it spread along the code. But that's just my opinion :)

Reply:
Services gets imported to  other projects wmarchive, crab, I was afraid break anything due to dependency. But I think it is OK for this.
https://github.com/dmwm/WMCore/blob/master/setup_dependencies.py
I will move the import up